src/App.js

```diff
@@ -20,6 +21,24 @@ function App() {
         >
           Learn React
         </a>
+
+        {/*download button, redownloads download icon on click*/}
+        <Button>
+        <a href='download_icon.png' download>
+        <img src="download_icon.png" alt=""/>
+        </a>
+        </Button>
+
+        {/*preview button, redirects to project GitHub page*/}
+        <br></br>
+        <Button
+        buttonStyle="btn--preview--solid">
+          <a href="https://github.com/the-template-project/root">
+          <img src="preview_icon.png" alt=""/>
+          </a>
+        </Button>
```

Comment:
great job with the download and preview buttons. Could you use those as examples in `App.js` instead of this?

Reply:
I think everything that's in the App div is going to be in the **<Homepage />** tag. Therefore, I only kept them as examples in the **button.jsx** . I thought it would be better because the examples could just be copied and pasted from the **button.jsx** file, making them more easily accessible. I saw it as, if I were to put them into the **App.js** file directly, it would not properly format into the homepage, it would still appear on screen and be functional, but would only format around the homepage rather than inside of it.

Reply:
Sorry, I meant instead of a <Button> elem to use a <PreviewButton> elem. It'll save a lot of space in the file and be more relevant as those are the buttons we have a planned used for.